DoLahTest/unittest1.cpp

```diff
@@ -3,12 +3,14 @@
 
 #include "basic.h"
 
+#include "..\DoLah\Models\Task.h"
+#include "..\DoLah\Commands\Command.h"
```

Comment:
* use forward slash path.
* include header & references during compilation/linking instead of hard coding the relative path of inter-project references.